sci-electronics/systemc/systemc-2.3.4-r1.ebuild

```diff
@@ -24,11 +24,10 @@ fi
 
 LICENSE="Apache-2.0"
 SLOT="0"
-IUSE="debug static-libs test"
+IUSE="debug doc examples static-libs test"
+REQUIRED_USE="examples? ( doc )"
 RESTRICT="!test? ( test )"
 
-DOCS=(AUTHORS.md CONTRIBUTING.md INSTALL.md LICENSE NOTICE README.md RELEASENOTES)
```

Comment:
should `doc` even be a use flag when it installs files that `default_src_install` intends to install? (aside from `*.md` renames)

Reply:
> should `doc` even be a use flag when it installs files that `default_src_install` intends to install? (aside from `*.md` renames)

This package will install many pdf file into /usr/share/doc

```
/usr/share/doc/systemc-2.3.4-r1/examples/sysc/simple_bus/SLIDES.pdf
/usr/share/doc/systemc-2.3.4-r1/examples/tlm/at_ooo/docs/at_ooo_example.pdf
/usr/share/doc/systemc-2.3.4-r1/examples/tlm/lt_temporal_decouple/docs/lt_temporal_decouple_example.pdf
/usr/share/doc/systemc-2.3.4-r1/examples/tlm/at_4_phase/docs/at_4_phase_example.pdf
/usr/share/doc/systemc-2.3.4-r1/examples/tlm/lt_mixed_endian/docs/lt_mixed_endian_example.pdf
/usr/share/doc/systemc-2.3.4-r1/examples/tlm/lt/docs/lt_example.pdf
/usr/share/doc/systemc-2.3.4-r1/examples/tlm/lt_dmi/docs/lt_dmi_example.pdf
/usr/share/doc/systemc-2.3.4-r1/examples/tlm/at_1_phase/docs/at_1_phase_example.pdf
/usr/share/doc/systemc-2.3.4-r1/examples/tlm/lt_extension_mandatory/docs/lt_extension_mandatory_example.pdf
/usr/share/doc/systemc-2.3.4-r1/examples/tlm/at_2_phase/docs/at_2_phase_example.pdf
/usr/share/doc/systemc-2.3.4-r1/examples/tlm/at_extension_optional/docs/at_extension_optional_example.pdf
/usr/share/doc/systemc-2.3.4-r1/examples/tlm/at_mixed_targets/docs/at_mixed_targets_example.pdf
/usr/share/doc/systemc-2.3.4-r1/sysc/archived/UserGuide20.pdf
/usr/share/doc/systemc-2.3.4-r1/sysc/archived/WhitePaper20.pdf
/usr/share/doc/systemc-2.3.4-r1/sysc/archived/FuncSpec20.pdf
/usr/share/doc/systemc-2.3.4-r1/sysc/archived/SystemC_2_1_features.pdf
/usr/share/doc/systemc-2.3.4-r1/sysc/archived/SystemC_2_1_overview.pdf
/usr/share/doc/systemc-2.3.4-r1/sysc/SystemC_2011_New_Features.pdf
/usr/share/doc/systemc-2.3.4-r1/tlm/release/TLM_2_0_presentation.pdf
/usr/share/doc/systemc-2.3.4-r1/tlm/release/TLM_2_0_white_paper.pdf
/usr/share/doc/systemc-2.3.4-r1/tlm/release/TLM_2_0_requirements.pd
```

In older version https://github.com/gentoo/gentoo/blob/master/sci-electronics/systemc/systemc-2.3.1-r1.ebuild, it have doc USE flag, so I add this use back.